samples/HostingPlayground/GreeterOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace HostingPlayground
+{
+    public class GreeterOptions
+    {
+        public string Name { get; set; }
```

Comment:
If this is intended to be called by the model binder, then you don't need the setter.